spec/osut_tests_spec.rb

```diff
@@ -2937,18 +2937,18 @@ module M
     vtx << mini_attic_vtx[2]
     vtx << mini_attic_vtx[1]
     vtx << mini_attic_vtx[0]
-    vtx << mini_attic_vtx[3] # ?
+    vtx << mini_attic_vtx[3]
     vtx << vtx[3]
     expect(attic_floor.setVertices(vtx)).to be true
 
     # Generate (temporary) OSM & IDF:
     file = File.join(__dir__, "files/osms/out/miniX.osm")
     model.save(file, true)
 
-    file = File.join(__dir__, "files/osms/out/miniX.idf")
-    ft   = OpenStudio::EnergyPlus::ForwardTranslator.new
-    idf  = ft.translateModel(model)
-    idf.save(file, true)
```

Comment:
The updated [v3.7.0-rc1](https://github.com/NREL/OpenStudio/pull/5003#issuecomment-1785353538) (OpenStudio PR #[5003](https://github.com/NREL/OpenStudio/pull/5003) runs just fine locally - nice! The PR is failing as GitHub Actions is still pulling the original v3.7.0-rc1. Keeping the test as is for now; will re-test against eventual v3.7.0 release.